### PR TITLE
appveyor: enable http command test

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -120,6 +120,15 @@ test_script:
      --timeout 60
      --n-retries 2
      test\command\suite
+  - ruby test\command\bin\grntest
+     --groonga %FULL_GROONGA_INSTALL_FOLDER%\bin\groonga.exe
+     --base-directory test\command
+     --reporter mark
+     --n-workers 1
```
Comment (komainu8) [inline appveyor.yml]: In my opinion, `--n-workers` are multiple better.
Because we are easy to find problems.

Comment (kenhys) [reply appveyor.yml]: It is reasonable to change.

**Comment by kenhys** [comment]
@komainu8  I've rebased and fixed to use multiple worker.

**Comment by komainu8** [comment]
@kenhys 
The following test fails.
https://ci.appveyor.com/project/groonga/groonga/builds/23730567/job/ey22198bc9qwydew#L8235

**Comment by kenhys** [comment]
@komainu8 
Hmm, https://ci.appveyor.com/project/groonga/groonga/builds/23730567
some works, but not all.
So, It seems that multiple worker is not appropriate...

**Comment by kenhys** [comment]
I've rebased and reverted multiple worker commit.

**Comment by kenhys** [comment]
https://ci.appveyor.com/project/groonga/groonga/builds/23876216 indicates that it can't be enabled http test because of execution time limit. (1h over) :cry:

**Pull request closed without merging**